specs/fault-proof/index.md

```diff
@@ -444,10 +444,10 @@ The L2 Output is the preimage of a
 
 Requests the host to prepare all preimages used in the execution of the block at `<blocknumber>` height.
 
-#### `l2-account-proof <blocknumber_and_address>`
+#### `l2-account-proof <blockhash_and_address>`
 
-Requests the host send account proof for a certain block number and address. `<blocknumber_and_address>` is hex
-encoded: 8-byte BE block number + 20-byte address.
+Requests the host send account proof for a certain block number and address. `<blockhash_and_address>` is hex
```

Comment:
Looks like there should be 1 more text update here:
```suggestion
Requests the host send account proof for a certain block hash and address. `<blockhash_and_address>` is hex
```

Reply:
ugh not sure how I missed that. I'll open another PR :)

Reply:
https://github.com/ethereum-optimism/specs/pull/443